crypto/perlasm/x86_64-xlate.pl

```diff
@@ -109,14 +112,16 @@
 
 { package opcode;	# pick up opcodes
     sub re {
-	my	$self = shift;	# single instance in enough...
-	local	*line = shift;
-	undef	$ret;
+	my	$class = shift;
+	my	$self = {};
+	my	$line = shift;
```

Comment:
(Yeah, I think `my (...) = @_;` is more readable too.)

I'm happy to change it to whichever is preferred. I only did it this way to match `register::re` as the one example of a non-global object to mimic. Should I change that one too?